Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
Ugh, this did not fix the problem: https://gist.github.com/8ca5bad096d294d9626da1f1a51f61fb

Regardless of whether Xcode or the CLT is the active developer directory.